### DISCUSSION TITLE
Running on Fedora's bash terminal...

### DISCUSSION THREAD

**Discussion by kepeter** [tui-cs/Terminal.Gui#1814] [Q&A]
Anyone has experience with this?
It seems to has a lot of problems with mouse/keyboard event (they are repeated) and the visual rendering is a bit off...
I did checked it on an Ubuntu system via putty (that uses xterm) and looked just fine, but on my physical Fedora running bash it is next to unusable...
Did I missed somethin?

**Comment by tznind** [comment]
Hey,  Please can you try running the UICatalog app with `-usc`.  This sets the `Application.UseSystemConsole = true;` and means it will run with the NetDriver instead of ncurses.  Are you able to provide a gif of the issue you are having with visual rendering? (e.g. using peek)

Are you testing the `main` branch or using the NuGet package?

What is the contents of your $TERM variable when running in Fedora?

**Reply by kepeter** [reply]
* $TERM is xterm-256color
* Setting UseSystemConsole seems to be solving the problem, no problems I can recognize with that on
* I'm using NuGet package version 1.6.4
* What I thought is a keyboard problem is related to a double-exception in my code :-(
* The mouse problem is still exist with the CoursesDriver - I do not know how to capture it but what I see that the last mouse event (e.g. the click to open the menu) is repeated whenever I stop on an element, like folder in the file open dialog...

**Comment by BDisp** [comment]
I can confirm that with the most recent `Ubuntu` and `Debian` version the last mouse event is always repeated, but with the keyboard I didn't found that behavior but perhaps have the same issue. The solution was to use the previous version because is the `ncurses` that return that wrong events. To use the latest version do as @tznind said above.
